### DISCUSSION TITLE
Is there a way to add MusicBee to the media controls

### DISCUSSION THREAD

**Discussion by skyisthela** [amnweb/yasb#50] [Q&A] [answered]
Absolutely loving this, however I noticed the music player I use doesn't cause the media controls to show up on the status bar. It can be controlled by it but unless I have a program open that actually causes the widget to show up there's nothing. The wiki doesn't seem to have anything (that I am aware of) on this either.

**Comment by amnweb** [comment, marked as answer]
I think related to this https://github.com/amnweb/yasb/issues/26

**Reply by skyisthela** [reply]
oh shit this was the answer. sorry for the over sight lol. thank you for your time, absolutely loving this and thank you for reviving what was seemingly a dead project
